cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -90,6 +102,21 @@ class TowerVariableValue(models.Model):
         ),
     ]
 
+    @api.depends("value_char", "option_id", "variable_id.option_ids")
```

Comment:
Do we need to call this function when `value_char` is updated?

Reply:
i will remove it